services/user.py

```diff
@@ -0,0 +1,51 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> User:
+    return get_user_model().objects.create_user(
+        username=username,
+        password=password,
+        email=email or "",
+        first_name=first_name or "",
+        last_name=last_name or ""
+    )
+
+
+def get_user(user_id: int) -> User:
+    return get_user_model().objects.get(pk=user_id)
+
+
+def update_user(
+        user_id: int,
+        username: str = None,
+        password: str = None,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None,
+) -> None:
+    user = get_user_model().objects.get(pk=user_id)
```

Comment:
Consider handling the `DoesNotExist` exception for `get_user_model().objects.get(pk=user_id)` to avoid unhandled exceptions if the user ID is not found.